.circleci/config.yml

```diff
@@ -96,7 +99,6 @@ jobs:
       - checkout
       - *restore_cache
       - run: |
-          npm run build --workspace=packages/components
```

Comment:
These are no longer needed since the workspaces are built once as part of a postinstall script to ensure that everything is set up right after first install